habitat/plan.sh

```diff
@@ -57,9 +58,7 @@ do_verify() {
 
 do_setup_environment() {
   push_runtime_env GEM_PATH "${pkg_prefix}/vendor"
-
   set_runtime_env APPBUNDLER_ALLOW_RVM "true" # prevent appbundler from clearing out the carefully constructed runtime GEM_PATH
-  set_runtime_env SSL_CERT_FILE "$(pkg_path_for cacerts)/ssl/cert.pem"
```

Comment:
I see the export to `SSL_CERT_FILE` on line 71, but have we confirmed that this is redundant?

Reply:
Yeah, this remove it's runtime usage and line 71 is used at build time. I'm not sure we want to remove this unless we know for sure we do not need it.

Reply:
@smandaka  could you pls try to bring that change back ( not remove the SSL_CERT_FILE) line from plan file and then try ?